tests/system/feature_store/test_feature_store.py

```diff
@@ -70,6 +71,38 @@ def _generate_random_name():
     return random_name
 
 
+kafka_brokers = os.getenv("KAFKA_BROKERS", "")
+
+kafka_topic = "kafka_integration_test"
+
+
+@pytest.fixture()
+def kafka_topic_setup_teardown():
```

Comment:
Since this will be the name used by tests when utilizing this fixture, I don't see the point in the `setup_teardown` name. `kafka_consumer` would be a better name since this is what the clients are actually getting.